### PR TITLE
ENH: update lag monitoring and a few plc soft limits

### PR DESCRIPTION
**Pull request by nrwslac**
## Description

- Reduce lag monitoring to 50 um for MR1L3, MR1L4, MR2L3 X Axes.
- Reduce lag monitoring for MR1L3, MR1L4 Pitch to 30 urad
- Reduce lag monitoring for MR2L3 Pitch to 15 urad
- Increase PLC Soft Limit of MR1L4 X from 18500 to 18750
- Decrease PLC Soft Limit of MR1L4 Pitch from 1000 to 900.
- Independent project files for MR1L4 and MR2L3 seems to have never been done; saved them.

## Motivation and Context

- ME Hardstop check.

## How Has This Been Tested?

- Running on PLC

## Where Has This Been Documented?

https://confluence.slac.stanford.edu/pages/viewpage.action?pageId=445874583
https://confluence.slac.stanford.edu/pages/viewpage.action?pageId=445154253
https://confluence.slac.stanford.edu/pages/viewpage.action?pageId=445874554

## Pre-merge checklist
- [ ] Code works interactively
- [ ] Code contains descriptive comments
- [ ] Test suite passes locally
- [ ] Libraries are set to fixed versions and not ``Always Newest``
- [ ] Code committed with ``pre-commit`` (alternatively ``pre-commit run --all-files``)

### CODE REVIEW

**Comment by joshc-slac** [comment]
I trust the vision and understand the need. Struggling to parse all these XML files would you be able to github permalink a representative selection where you set this? And or screen shot from where I am pretty confident you set this in the twincat ide.

**Review by joshc-slac — APPROVED** [review]
LGTM (so much xml)

File: HOMS_XRT/_Config/NC/NC.xti [hunk 1]
```diff
@@ -2285,7 +2285,7 @@ External Setpoint Generation:
 			</Drive>
 			<Controller Name="Ctrl" CtrType="2">
 				<CtrPara PriorControlFactor="1">
-					<PosDiffControl Range="500"/>
```
Comment (nrwslac) [inline HOMS_XRT/_Config/NC/NC.xti:2288]: The NC.xti looks like the easiest spot to look for changes @joshc-slac

File: HOMS_XRT/_Config/NC/Axes/M1L3-Pitch.xti [hunk 2]
```diff
@@ -1493,7 +1467,7 @@ Drive Status 4 (manually linked):
 		</Drive>
 		<Controller Name="Ctrl" CtrType="2">
 			<CtrPara PriorControlFactor="1">
-				<PosDiffControl Range="200"/>
```
Comment (nrwslac) [inline HOMS_XRT/_Config/NC/Axes/M1L3-Pitch.xti:1496]: lag monitoring

File: HOMS_XRT/_Config/NC/Axes/M1L3-Xdwn.xti [hunk 3]
```diff
@@ -1492,7 +1466,7 @@ Drive Status 4 (manually linked):
 		</Drive>
 		<Controller Name="Ctrl" CtrType="2">
 			<CtrPara PriorControlFactor="1">
-				<PosDiffControl Range="100"/>
```
Comment (nrwslac) [inline HOMS_XRT/_Config/NC/Axes/M1L3-Xdwn.xti:1495]: lag monitoring.

**Comment by nrwslac** [comment]
We need to rebuild the IOC and release it whens its good timing. I guess tomorrow is a POM?

**Comment by joshc-slac** [comment]
> We need to rebuild the IOC and release it whens its good timing. I guess tomorrow is a POM?

will DM, in short yes

**Pull request merged**